stable/provazio/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.controller.enabled }}
+
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "provazio.controller.name" . }}
+  labels:
+    app: {{ template "provazio.controller.name" . }}
+    chart: {{ template "provazio.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: 1
+  selector:
```

Comment:
I'm not sure we require a selector. Can be removed.

Reply:
As far as I can tell this is required in apps/v1